sources/packages/web/src/components/ValidatedInput.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div>
+    <div :class="{ 'validation-error': property.$invalid }">
+      <slot></slot>
```

Comment:
I thing slot is deprecated and v-slot is introduced. 
https://vuejs.org/v2/guide/components-slots.html#Deprecated-Syntax
Can you check

Reply:
Yes, that makes sense. I will change it.

Reply:
Hi @fwpushan, the page that you mentioned is related to Vue 2 where it states the new way to do it in Vue 2 in comparison with the previous version.
The slots on Vue 3 are to be used it is in the code right now.
https://v3.vuejs.org/guide/component-slots.html#slot-content